src/generic/MPoly.jl

```diff
@@ -1387,10 +1387,6 @@ struct nheap_t
    next::Int
 end
 
-function ==(a::heap_s, b::heap_s)
-   return exps[a.exp] == exps[b.exp]
-end
-
```

Comment:
Why remove this? is it never used?

Reply:
Apparently as otherwise an error would be reported as exps is undefined

Reply:
ah ok